### DISCUSSION TITLE
Pore network consisting only of tubes

### DISCUSSION THREAD

**Discussion by LukasPore** [PMEAL/OpenPNM#2366] [Q&A]
Hello,

I have a question regarding the generation of a porenetwork in OpenPNM. 
Is it possible to generate a porenetwork consisting only of tubes with different diameters?
In addition, I'm wondering if it is possible to use different pore size distributions for the x and y or z direction in a cubic pore network?

Sincerely thanks for your help.
Lukas

**Comment by jgostick** [comment]
Hi @LukasPore 
1. A network *must* have "junctions' to define the start and end of each throat.  These do not need to have any physical size though, so if you just want throats, you could set the diameter of the pores to 0...BUT be careful when computing the transport conductance that you only use the throat information.  If you use one of openpnm's built in conductance models you will get 0 conductance in the conduit since the conductance of the 'pores' was set to 0.
2. Pores don't have a direction. If you're thinking about 'throat size distribution' then yes it's possible.  There is a model (``op.models.geometry.throat_vector.pore_to_pore``) that computes the vector of each throat. You could create 3 different geometries, once for each direction, and assign a different throat size distribution to each geometry.
